fearless/Common/Extension/Foundation/NumberFormatter.swift

```diff
@@ -24,11 +24,17 @@ extension NumberFormatter {
     static var percentAPY: NumberFormatter {
         let numberFormatter = NumberFormatter.percent
         numberFormatter.minimumFractionDigits = 2
-        numberFormatter.positivePrefix = ""
+        numberFormatter.positivePrefix = "+"
         numberFormatter.percentSymbol = "% APY"
         return numberFormatter
     }
 
+    static var percentAPR: NumberFormatter {
```

Comment:
I suggest not to reuse APY implementation and just setup formatter from scratch

Reply:
Okay, what about reusing percent implementation?

Reply:
Not great fan of it but at least it will be consistent with APY